RELEASE_NOTES.md

```diff
@@ -1,3 +1,8 @@
+#### 3.0.0-beta - February 27 2018
+* Drop SourceLink support
```

Comment:
No longer true, right?

Reply:
Correct. I added it back in, and it seamlessly happens during the build step.